packages/kaizen-tg/_templates/drupal-9-theme/new/.stylelintignore

```diff
@@ -2,3 +2,6 @@
 to: <%= h.src() %>/<%= h.changeCase.lower(name) %>/.stylelintignore
 ---
 /color/*.css
+node_modules/**/*.css
```

Comment:
This seems useless, because stylelint already ignoring node_module's css (at least it should from what i know)

Reply:
Ok. I will remove this.